robotidy/transformers/GenerateDocumentation.py

```diff
@@ -0,0 +1,187 @@
+import re
+from pathlib import Path
+from typing import Optional
+
+from jinja2 import Template
+from robot.api.parsing import Documentation, ModelVisitor, Token
+
+from robotidy.exceptions import InvalidParameterValueError
+from robotidy.transformers import Transformer
+
+GOOGLE_TEMPLATE = """
+{% if keyword.arguments|length > 0 %}
+{{ formatting.cont_indent }}Arguments:
+{%- for arg in keyword.arguments %}
+{{ formatting.cont_indent }}{{ formatting.cont_indent }}{{ arg.name }}: {% endfor %}
+{% endif -%}
+{% if keyword.returns|length > 0 %}
+{{ formatting.cont_indent }}Returns:
+{%- for value in keyword.returns %}
+{{ formatting.cont_indent }}{{ formatting.cont_indent }}{{ value }}: {% endfor %}
+{% endif -%}
+"""
+
+
+class Argument:
+    def __init__(self, arg):
+        if "=" in arg:
+            self.name, self.default = arg.split("=", 1)
+        else:
+            self.name = arg
+            self.default = None
+        self.full_name = arg
+
+    def __str__(self):
+        return self.full_name
+
+
+class KeywordData:
+    def __init__(self, name, arguments, returns):
+        self.name = name
+        self.arguments = arguments
+        self.returns = returns
+
+
+class FormattingData:
+    def __init__(self, cont_indent, separator):
+        self.cont_indent = cont_indent
+        self.separator = separator
+
+
+class ArgumentsAndsReturnsVisitor(ModelVisitor):
```

Comment:
```suggestion
class ArgumentsAndReturnsVisitor(ModelVisitor):
```

Reply:
Took me solid 30s to find the difference. Thanks!